docs/running.md

```diff
@@ -9,12 +9,110 @@ a file `$XDG_STATE_HOME/warnet/warnet.log`, otherwise it will use `$HOME/.warnet
 
 ## Kubernetes
 
-Deploy the resources in `src/templates/`, this sets up the correct permissions on the cluster (`rbac-config.yaml`) and deploys the warnet RPC server as a service + statefulset.
+Deploy the resources in `src/templates/`, this sets up a Kubernetes cluster, apply correct permissions on the cluster (`rbac-config.yaml`), and deploys the warnet RPC server as a service + statefulset.
 
-This can be done with from inside the `src/templates/` directory by running:
+Below are the commands to set these up correctly:
 
 ```bash
-kubectl apply -f '*.yaml'
+# Creates a Kubernetes namespace for the RPC
+kubectl apply -f src/templates/rpc/namespace.yaml
+
+# Applies Role-Based Access Control (RBAC) configuration for the RPC
+kubectl apply -f src/templates/rpc/rbac-config.yaml
+
+# Deploys a service to expose the RPC
+kubectl apply -f src/templates/rpc/warnet-rpc-service.yaml
+
+# Deploys a statefulset for the RPC to manage stateful applications
+kubectl apply -f src/templates/rpc/warnet-rpc-statefulset-dev.yaml
```

Comment:
the `-dev.yaml` is for local development (changes in the warnet codebase are reflected in the container) so we can probably just chop that off and illustrate use of the production image which is just `warnet-rpc-statefulset.yaml`. That applies to this entire document.

Reply:
Ok. I'll update that.